### PR TITLE
Added Webupload Module

### PR DESCRIPTION
**Pull request by shivamvijaywargi**
Check this if it's correct.

### CODE REVIEW

File: userge/plugins/misc/webupload.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#  Thanks to Userge team for making this userbot
+# Special thanks to Gotstc for helping me out.
+#  Atlast to me for doing almost nothing LOL
```
Comment (rking32) [inline userge/plugins/misc/webupload.py:3]: please remove these lines and add license text
```suggestion
# Copyright (C) 2020 by UsergeTeam@Github, < https://github.com/UsergeTeam >.
#
# This file is part of < https://github.com/UsergeTeam/Userge > project,
# and is released under the "GNU v3.0 License Agreement".
# Please see < https://github.com/uaudith/Userge/blob/master/LICENSE >
#
# All rights reserved.
```

File: userge/plugins/misc/webupload.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        await message.err("Invalid selected Transfer")
+        return
+    cmd = selected_one
+
+    # process = await asyncio.create_subprocess_shell(
+    #    cmd, stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE
+    # )
+    # stdout, stderr = await process.communicate()
+    # await message.edit(f"{stdout.decode()}")
+    response = subprocess.check_output(shlex.split(cmd))
```
Comment (rking32) [inline userge/plugins/misc/webupload.py:57]:
```suggestion
    process = await asyncio.create_subprocess_exec(
       *shlex.split(cmd), stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE
    )
    response, _ = await process.communicate()
```

File: userge/plugins/misc/webupload.py (hunk 1 above)
Comment (shivamvijaywargi) [reply userge/plugins/misc/webupload.py]: Done, I guess.

File: userge/plugins/misc/webupload.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+# All rights reserved.
+
+
+import os
+import io
+import re
+import shlex
+import time
+import asyncio
+import subprocess
```
Comment (rking32) [inline userge/plugins/misc/webupload.py:16]: clean

File: userge/plugins/misc/webupload.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        "bayfiles": "curl -F \"file=@{}\" https://api.bayfiles.com/upload"
+    }
+    try:
+        selected_one = CMD_WEB[selected_transfer].format(
+            file_name)
+    except KeyError:
+        await message.err("Invalid selected Transfer")
+        return
+    cmd = selected_one
+
```
Comment (rking32) [inline userge/plugins/misc/webupload.py:51]:
```suggestion
```

File: userge/plugins/misc/webupload.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    process = await asyncio.create_subprocess_exec(
+       *shlex.split(cmd), stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE
+    )
+    response, _ = await process.communicate()
+    links = '\n'.join(re.findall(r'https?://[^\"\']+', response.decode()))
+    print(response, links)
```
Comment (rking32) [inline userge/plugins/misc/webupload.py:57]:
```suggestion
```

**Comment by rking32** [comment]
whats going on, did you complete this ?
if yes, we can merge it to dev

**Comment by rking32** [comment]
moved to https://github.com/UsergeTeam/Userge-Plugins

**Pull request closed without merging**